app/components/Form/ProjectFundingAgreementForm.tsx

```diff
@@ -118,19 +123,62 @@ const ProjectFundingAgreementForm: React.FC<Props> = (props) => {
   const handleError = () => {
     handleChange(fundingAgreement.newFormData, "staged");
   };
+  const [showDiscardConfirmation, setShowDiscardConfirmation] = useState(false);
 
+  const handleDiscard = () => {
+    setShowDiscardConfirmation(false);
+    discardFundingParameterFormChange({
+      variables: {
+        input: {
+          revisionId: projectRevision.rowId,
+        },
+        connections: [],
+        reportType: "funding_parameter",
+      },
+    });
+  };
   return (
     <>
       {projectRevision.projectFundingAgreementFormChanges.edges.length ===
         0 && (
-        <Button onClick={addFundingAgreement} style={{ marginRight: "1rem" }}>
-          Add Funding Agreement
-        </Button>
+        <div>
+          <h3>Is this a funded project?</h3>
+          <RadioButton
+            name="yes"
+            label="Yes"
+            className="radio-button"
+            onClick={addFundingAgreement}
+          />
+          <RadioButton name="yes" label="No" className="radio-button" />
+        </div>
       )}
       {projectRevision.projectFundingAgreementFormChanges.edges.length > 0 && (
         <>
+          <div>
+            <h3>Is this a funded project?</h3>
+            <RadioButton
+              name="yes"
+              label="Yes"
+              className="radio-button"
+              checked={true}
+            />
+            <RadioButton
+              name="yes"
+              label="No"
+              className="radio-button"
+              onClick={() => setShowDiscardConfirmation(true)}
+            />
+            {showDiscardConfirmation && (
+              <Alert variant="danger" size="sm">
+                All changes made will be deleted.
+                <a onClick={handleDiscard} id="confirm-discard-revision">
+                  Proceed
+                </a>
+                <a onClick={() => setShowDiscardConfirmation(false)}>Cancel</a>
+              </Alert>
```

Comment:
This should be refactored in its own component, we have the exact same code and styles in `[projectRevision]/index.tsx`